src/test/java/seedu/address/logic/commands/CleanCommandTest.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CleanCommand.MESSAGE_CLEAN_SUCCESS;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
+
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code CleanCommand}.
+ */
+public class CleanCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void execute_clean_success() {
```

Comment:
You are not supplying a predicate in this test case, so expectedModel always has an empty address book?

I am assuming that you will add the predicate after clean command is implemented.

Reply:
Any predicate needed? Clean Command creates its own predicate, and expected model has deleted the graduated people.